extensions/adobe/experience/marketo/marketo-person-contact.schema.json

```diff
@@ -0,0 +1,204 @@
+          "title": "Additional Fields",
+          "description": "Additional fields.",
+          "type": "string"
+        },
+        "marketo:anonymousHostname": {
+          "title": "Anonymous Hostname",
+          "description": "Anonymous hostname.",
+          "type": "string"
+        },
+        "marketo:blackListed": {
+          "title": "Black Listed Indicator",
+          "description": "Black listed indicator.",
+          "type": "boolean"
+        },
+        "marketo:blackListedCause": {
+          "title": "Black Listed Cause",
+          "description": "Black listed cause.",
+          "type": "string"
+        },
+        "marketo:emailInvalid": {
+          "title": "Invalid Email Indicator",
+          "description": "Invalid email indicator.",
+          "type": "boolean"
+        },
+        "marketo:emailInvalidCause": {
+          "title": "Email Invalid Cause",
+          "description": "Email invalid cause.",
+          "type": "string"
+        },
+        "marketo:employeePositionID": {
+          "title": "Employee Position Id",
+          "description": "Employee position id.",
+          "type": "string"
+        },
+        "marketo:inferredAddress": {
+          "title": "Inferred Address",
+          "description": "Address details inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "$ref": "https://ns.adobe.com/xdm/common/address"
+        },
+        "marketo:inferredCompany": {
+          "title": "Inferred Company",
+          "description": "Company name inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredCompanyStatus": {
+          "title": "Inferred Company Status",
+          "description": "Status of the company name inferred by reverse IP lookup of the lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredMetropolitanArea": {
+          "title": "Inferred Metropolitan Area",
+          "description": "Metropolitan area inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:inferredPhoneAreaCode": {
+          "title": "Inferred Phone Area Code",
+          "description": "Phone area code inferred by reverse IP lookup of lead’s first recorded web visit.",
+          "type": "string"
+        },
+        "marketo:anonymous": {
+          "title": "Anonymous Indicator",
+          "description": "Anonymous status of lead record. System managed.",
+          "type": "boolean"
+        },
+        "marketo:customer": {
+          "title": "Customer Indicator",
+          "description": "Customer indicator.",
+          "type": "boolean"
+        },
+        "marketo:employee": {
+          "title": "Employee Indicator",
+          "description": "Employee indicator.",
+          "type": "boolean"
+        },
+        "marketo:lead": {
```

Comment:
Not Required, we already have leadSource in profile-crm-mixin

Reply:
Will remove